verify/install-service

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+set -eufx
+
+SECRETS=/opt/cockpit-verify-secrets
+IMAGES=/opt/cockpit-verify-images
+
+echo "options kvm-intel nested=1" > /etc/modprobe.d/kvm-intel.conf
+echo "options kvm-amd nested=1" > /etc/modprobe.d/kvm-amd.conf
+( rmmod kvm-intel && modprobe kvm-intel ) || ( rmmod kvm-amd && modprobe kvm-amd )
```

Comment:
Can this be guarded with `|| true`, in case `/dev/kvm` or the kernel modules are not available?